packages/ariakit/src/tooltip/__examples__/tooltip/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { Tooltip, TooltipAnchor, useTooltipState } from "ariakit/tooltip";
+
+export default function Example() {
+  const tooltip = useTooltipState();
+  return (
+    <>
+      <TooltipAnchor state={tooltip} as="button">
+        Hover on me!
+      </TooltipAnchor>
+      <Tooltip state={tooltip}>You hovered on a button</Tooltip>
+    </>
+  );
+}
```

Comment:
Here's a suggestion. `TooltipAnchor` is a component that should be used in combination with others. It's okay to render it `as="button"` as I'd suggested, but I think we should recommend using our `Button` instead.

Also added class names so we can style them.

```suggestion
import { Button } from "ariakit/button";
import { Tooltip, TooltipAnchor, useTooltipState } from "ariakit/tooltip";
import "./style.css";

export default function Example() {
  const tooltip = useTooltipState();
  return (
    <>
      <TooltipAnchor state={tooltip} as={Button} className="button">
        Hover or focus on me
      </TooltipAnchor>
      <Tooltip state={tooltip} className="tooltip">
        Tooltip
      </Tooltip>
    </>
  );
}

```